nextflow_schema.json

```diff
@@ -630,5 +630,343 @@
         {
             "$ref": "#/definitions/institutional_config_options"
         }
-    ]
-}
+    ],
+    "properties": {
+        "enable_ms1": {
```

Comment:
Please add at least a `description` to every single parameter. Preferably an icon too.

The point of the schema docs is to make the parameters accessible and easy to understand. If they are just a huge list of names without any customisation or documentation then that is fairly unusable as an end-user.

Reply:
Note that it will soon be a hard requirement for every schema parameter to have a description, meaning that the nf-core lint tests will fail without it.